docs/specs/shapes.md

```diff
@@ -477,6 +477,8 @@ A stroke dash array consists of `n` dash entries, `[n-1,n]` gap entries and `[0-
 
 Dash and gap entries MUST all be in a continuous order and alternate between dash and gap, starting with dash. If there are an odd number of dashes + gaps, the sequence will repeat with dashes and gaps reversed. For example a sequence of `[4d, 8g, 16d]` MUST be rendered as `[4d, 8g, 16d, 4g, 8d, 16g]`.
 
+If any of the dash or gap entries are less than 0, or if the total sum of all their values is 0, the whole dash array is ignored, and the stroke should be treated as if the dash was not set.
```

Comment:
> should
We're using [rfc2119](https://www.rfc-editor.org/rfc/rfc2119) use the correct words and ensure they are upper case. In this case I guess "MUST" shall be used instead of "should".

Reply:
changed